docs/app/live-editor.js

```diff
@@ -126,6 +127,13 @@ export default function LiveEditor() {
   }))
   const debouncedTokenize = debouncedTokenizeRef.current
 
+  const customizableColorsString=useMemo(()=>{
```

Comment:
need to prettier the changes

Reply:
Can you share your Prettier configuration? My configuration should be different from yours.

Reply:
Or it’s good to just leave the space between the operators like others. I didn’t prettify it but prefer the consistent code style

Reply:
I did that just now, do you see if there are any other issues?